schemas/vscs.json

```diff
@@ -550,7 +550,6 @@
                 "default": false,
                 "description": "This will override the Metro Provided config on the VSC by the config provided in vsc_config_file_paths",
                 "advanced": true,
-                "sectionEnd": "VSC Options",
```

Comment:
Why was this removed?

Reply:
Sanket sir was making some changes in schemas but later we did not followed this approach, i guess it must be removed by mistake.

Reply:
Ok, let's add the sectionEnd back. Then run the GENERATE-ALL-FROM-SCHEMAS Jenkins job to update the deployment templates

Reply:
ok sure.